bom/application/pom.xml

```diff
@@ -1514,6 +1514,12 @@
                 <version>2.6</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.wiremock</groupId>
+                <artifactId>wiremock</artifactId>
+                <version>3.5.3</version>
+            </dependency>
+
```

Comment:
@spbolton please review this requirement if we need it and with an eye to our java 21 migration

Reply:
If there are problems it may be due to conflicts in servlet container it uses jetty I think internally,  if there are issues also we can use wiremock-standalone which is a self contained shaded jar.  I don't think there are specific issues here with reflection.  It is not mocking classes with reflection like PowerMock etc